src/Symfony/Component/DependencyInjection/Exception/EnvParameterException.php

```diff
@@ -18,8 +18,8 @@
  */
 class EnvParameterException extends InvalidArgumentException
 {
-    public function __construct(array $usedEnvs, \Exception $previous = null)
+    public function __construct(array $envs, $message, \Exception $previous = null)
```

Comment:
in order to not break BC, the $message arg should be added last, and should be optional ($message = 'Incompatible...')